src/modules/Expanders.jl

```diff
@@ -272,6 +272,104 @@ immutable DocsNodes
     nodes :: Vector{DocsNode}
 end
 
+"""
+Converts the function argument tuple `(name, type)` into a string.
+
+The tuple comes from the second return element of the `Base.arg_decl_parts(::Method)`
+and it seems they are always both `::String` (`::ASCIIString` in 0.4).
+It also appears that if the type is not declared for the method, `arg_decl_parts`
+returns an empty string.
+
+The returned string is `name::type` or just `name`, if the type is not declared.
+"""
+function _function_arg(decl)
+    n, t = decl
+    isempty(t) ? n : "$(n)::$(t)"
+end
+
+"""
+Returns a markdown paragraph (`Markdown.Paragraph`) with a list of methods.
+
+The return value will be wrapped in a `Nullable`, and a `null` is returned
+if the object in `ex` has no methods.
+"""
+function methodtable_md(mod::Module, object::Utilities.Object, page, doc)
+    # In case the object is a method (e.g. given as `foo(x,y)` in the @docs
+    # block) we do not generate a methods table.
+    if Utilities.doccat(object) == "Method"
+        return Nullable{Markdown.Paragraph}()
+    end
+
+    ms = try
+        methods(eval(object.binding.mod, object.binding.var))
+    catch e
+        warn("Unable to eval $(object)")
+        info("Error: $e")
+        nothing
+    end
+
+    # If we were not able to eval the expression or the object doesn't have any
+    # methods (such as modules), we just return a `null`.
+    if ms === nothing || isempty(ms)
+      return Nullable{Markdown.Paragraph}()
+    end
+
+    pkgdir = Pkg.dir()
+
+    s = IOBuffer()
+    print(s, """
+    <p><strong>Methods</strong></p>
+    <ol class="documenter-methodtable">
+    """)
+
+    # Determine the name of the method.
+    # In general the name is Module.Submodule.object, and so we drop all
+    # the modules and just pick the name.
+    name = string(object.binding.var)
```

Comment:
`string` not really needed since it's only used in the two string interpolations below.